### PR TITLE
The great migration

### PR DESCRIPTION
**Pull request by nickmwangemi**
NamibLII
image

SeyLII
image

LesothoLII
image

SierraLII
image

ZimLII
image

TCILII
image

TanzLII
image

BotswanaLII
image

SwaziLII
image

### CODE REVIEW

**Comment by actlikewill** [comment]
@nickmwangemi I can see some crocodiles are attacking the herd as they cross the river :laughing:

**Comment by nickmwangemi** [comment]
> @nickmwangemi I can see some crocodiles are attacking the herd as they cross the river laughing

Still a WIP

File: sierralii/static/images/banner.jpg [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/nickmwangemi/Downloads/banner.jpg
```
Comment (longhotsummer) [inline sierralii/static/images/banner.jpg:1]: Looks like a symlink :)

**Review by longhotsummer — CHANGES_REQUESTED** [review]
Nice!

* The sierralii banner image needs to be added
* the lawlibrary.org.za banner image isn't showing up (i think it needs to be renamed)

File: sierralii/static/images/banner.jpg (hunk 1 above)
Comment (actlikewill) [reply sierralii/static/images/banner.jpg]: @longhotsummer Changed

**Review by longhotsummer — DISMISSED** [review]

**Pull request merged**